specification/trace/semantic_conventions/messaging.md

```diff
@@ -80,6 +80,48 @@ Often such destinations are unnamed or have an auto-generated name.
 
 Given these definitions, the remainder of this section describes the semantic conventions for Spans describing interactions with messaging systems.
 
+### Context propagation
+
+A message may pass many different components and layers in one or more intermediaries
+when it is propagated from the producer to the consumer(s). To be able to correlate
+consumer traces with producer traces using the existing context propagation mechanisms,
+all components must propagate context down the chain.
+
+This imposes a difficulty because it cannot be assumed, and in many cases,
```

Comment:
This is a very long and nested sentence. It's a bit hard to follow for me. I'd prefer having more, but shorter sentences.

Reply:
I agree. What you think about this:

```
This imposes a difficulty because it requires that all components are instrumented
and propagate context according to OpenTelemetry requirements. In reality,
this cannot be assumed, and in many cases is not even desired nor possible.
```

Reply:
Or 

```
This imposes a requirement that all components are instrumented
and propagate context according to OpenTelemetry requirements. In reality,
this cannot be assumed, and in many cases is not even desired nor possible.
```

Reply:
Went with the last in [4d89927](https://github.com/dynatrace-oss-contrib/opentelemetry-specification/pull/22/commits/4d89927be29348fe8a06aee2193b170a3b6e7fc1). Let me know if it's better!

Reply:
Started a new conversation in https://github.com/dynatrace-oss-contrib/opentelemetry-specification/pull/22#discussion_r951387777. Still feels a bit clunky to me.